client/src/components/SimpleDataField.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: ${(props) => props.theme.colors.input};
+  border-radius: 1.25rem;
+  border: none;
+  padding: 0.6rem;
+  outline: none;
+`;
+
+const Headline = styled.input`
+background-color: ${(props) => props.theme.colors.input};
+color: ${(props) => props.theme.colors.textSecondary}
+font-weight: bold;
+outline: none;
+cursor: pointer;
+max-width: 100%;
```

Comment:
Currently, the `size={65}` is setting a fix width bigger than your `Textareacontainer` for the input but the `max-width: 100%` will then reduce it to fit into the Container. The same happens with the textarea. 

The suggestions is to remove the fix width properties like `size={65}` from the input and `cols={60}` from the textarea and replace 'max-width: 100%' with `width: 100%`

```suggestion
width: 100%;
```

Reply:
You are absolutly right! Thanks!